libs/common-transfers/src/lib/components/excluded-dates.component.ts

```diff
@@ -23,15 +25,15 @@ export class ExcludedDatesComponent implements PaymentFormField, OnInit, OnDestr
   control!: AbstractControl;
   validationControl!: FormControl;
 
-  validationLabel = 'Start on';
+  validationLabel = 'Date';
   validationMessages = [
     { name: 'excludedDate', message: 'Cannot transfer on this day' },
     {
-      name: 'pastDate',
+      name: 'dateLessThanMinDate',
```

Comment:
Rename to the validation applied by OOB validators - see related comment in the service file.